packages/get-platform/src/__tests__/parseOpenSSLVersion.test.ts

```diff
@@ -72,6 +72,11 @@ describe('parseLibSSLVersion', () => {
       content: `/lib/libssl.so.3.1`,
       expect: '3.0.x',
     },
+    {
+      name: 'undefined',
+      content: `/lib/libssl3.so`,
+      expect: undefined,
+    },
```

Comment:
Random missing test case I had in the drafts locally

Reply:
maybe also add a comment

```ts
// nss library must not be mistakenly parsed as openssl
```

Reply:
I'll add it as the `name` value directly